appeals/web/src/server/lib/page-components/document.js

```diff
@@ -0,0 +1,64 @@
+import { isFolderInfo } from '#lib/ts-utilities.js';
+import { mapActionComponent } from '#lib/mappers/component-permissions.mapper.js';
+import { formatDocumentActionLink, formatDocumentValues } from '#lib/display-page-formatter.js';
+
+/**
+ * Returns the instructions for a document display field
+ *
+ * @param {Object} options
+ * @param {string} options.id
+ * @param {string} options.text
+ * @param {number} options.appealId
+ * @param {import('@pins/appeals.api').Appeals.FolderInfo|null|undefined} options.folderInfo
+ * @param {string} options.requiredPermissionName
+ * @param {import('../../app/auth/auth-session.service.js').SessionWithAuth} options.session
+ * @param {string} options.manageUrl
+ * @param {string} options.uploadUrlTemplate
+ * @param {string} [options.cypressDataName]
+ * @returns {Instructions}
+ */
+export function documentTypeDisplayField({
```

Comment:
Again I think `field` is too generic here. This is returning a summary list item display instruction, ideally the name would reflect that

Reply:
Gone with `documentTypeDisplayInstruction`